RentACarWebAPI/Controllers/CarController.cs

```diff
@@ -40,9 +41,9 @@ public IActionResult Get()
         [HttpGet("{id}")]
         [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(CarDto))]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
-        public IActionResult Get(int id)
+        public async Task<IActionResult> GetAsync(int id)
```

Comment:
Should i now response to a 202 code status? (same for the other methods that have "Ok" responses)

Reply:
why would you return a 202?

Reply:
Now that you ask, im confusing concepts. I thought that because it's a promise (async), but now that i write this, i realize that its an internal question, and in the response it will always return the completed task.

Reply:
exactly! (but we aware! this is not a promise)

Reply:
Understood! Thanks!! 😃